products-documentation-ingestion/milvus_handler.py

```diff
@@ -123,7 +123,13 @@ def ingest_documentation(self, collection, version, chunk_size=768, chunk_overla
             f"Calculating embeddings and uploading documents to collection {collection_name}"
         )
 
-        vector_store.add_documents(splits, batch_size=1)
+        # Process in batches of 600
+        total_splits = len(splits)
+        for i in range(0, total_splits, batch_size):
+            end_idx = min(i + batch_size, total_splits)
+            current_batch = splits[i:end_idx]
+            print(f"Processing batch {i//batch_size + 1}/{(total_splits + batch_size - 1)//batch_size}: documents {i} to {end_idx-1}")
+            vector_store.add_documents(current_batch, batch_size=self.milvus_batch_size)
```

Comment:
You changed from 1 to milvus_batch_size (which is 16), good update to add it as a var but I'm not sure what impact going from 1->16 will have

Reply:
As I commented above, to be tested to find a suitable value, set in the config.
Imho the, default should stay at 1, not 600. Highly inefficient, but failsafe.

Reply:
Cansu wrote it for another comment, different batch sizes, the one you are thinking of is set to 1 still :)